modules/lightning_features/lightning_media/lightning_media.install

```diff
@@ -278,16 +279,24 @@ function lightning_media_update_8016() {
 }
 
 /**
- * Implements hook_update_dependencies().
+ * Changes all image_immutable field widgets to image_image.
  */
-function lightning_media_update_dependencies() {
-  return [
-    'lightning_media' => [
-      // 8012 adds the field_media_in_library filter to the Entity Browser
-      // display of the media view, which is created by 8008.
-      8012 => [
-        'lightning_media' => 8008,
-      ],
-    ],
-  ];
+function lightning_media_update_8017() {
+  \Drupal::service('plugin.manager.field.widget')->clearCachedDefinitions();
+
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+  foreach (EntityFormDisplay::loadMultiple() as $form_display) {
+    foreach ($form_display->getComponents() as $key => $component) {
+      if ($component['type'] == 'image_immutable') {
+        $component['third_party_settings']['lightning_media'] = [
```

Comment:
We no longer need the image_immutable widget -- we can achieve the same effect in a more streamlined fashion by using third-party widget settings. This migrates from image_immutable to those.